### DISCUSSION TITLE
flink 1.13 error

### DISCUSSION THREAD

**Discussion by xhmz** [apache/doris#6623] [RFC]
switched from RUNNING to FAILED with failure cause: java.lang.ClassCastException: org.apache.flink.table.data.binary.BinaryRowData cannot be cast to org.apache.flink.table.data.GenericRowData
	at org.apache.doris.flink.table.DorisDynamicOutputFormat.addBatch(DorisDynamicOutputFormat.java:142)

**Comment by hf200012** [comment]
Please provide more information
